src/dataQuery.tsx

```diff
@@ -246,7 +247,7 @@ export function LoadNode({
   const filterArray = filtersToFilterArray(filter);
   const { knowledgeDBs, eose, allEventsProcessed } =
     useQueryKnowledgeData(filterArray);
-  if (waitForEose === true && !eose) {
+  if (isUserLoggedIn(user) && waitForEose === true && !eose) {
```

Comment:
we had some idea to let the user see notes from a default_user even if not logged in, how do we manage that?

Reply:
That's a separate Task. We will do that the same way we show the default workspace if the user has none by creating virtual events, which we then publish as soon as the user logs in.